### ISSUE TITLE
Build mdwiki from source: which versions of npm/node.js?

### ISSUE THREAD

**Issue by bidspells** [Dynalon/mdwiki#303]
Dear committers, 

I tried to build mdwiki from the sources on the latest release of MacOS - but failed. It seems to me that the problem is to choose the right npm / node.js versions. Using nvm I tried two combinations:

1. {"node":"0.10.32","npm":"1.4.28"} 
```
Using tsc v1.9.0-dev.20160627-1.0
js/ts/main.ts(15,7): error TS2339: Property 'initMDwiki' does not exist on type 'JQueryStatic'.

>> 1 non-emit-preventing type warning  
>> Error: tsc return code: 2
Warning: Task "ts:base" failed. Use --force to continue.

Aborted due to warnings.
make: *** [all] Error 3
```

2. node v6.9.1 (npm v3.10.8)

No errors in the build output it seems (see below), but mdwiki is not working properly (menus and navigation are damaged).mdwiki.html file is unusual small (ca. 268k).

I'm happy to receive some hints.

```
npm install
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated graceful-fs@1.1.14: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN prefer global coffee-script@1.10.0 should be installed with -g
npm WARN prefer global http-server@0.9.0 should be installed with -g

> fsevents@0.3.8 install /Users/kamsties/Code/mdwiki/node_modules/fsevents
> node-gyp rebuild

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  SOLINK_MODULE(target) Release/.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9
  CXX(target) Release/obj.target/fse/fsevents.o
./Release/.deps/Release/obj.target/fse/fsevents.o.d.raw { dev: 16777220,
  mode: 33188,
  nlink: 1,
  uid: 501,
  gid: 20,
  rdev: 0,
  blksize: 4096,
  ino: 33430434,
  size: 1386,
  blocks: 8,
  atime: 2016-11-22T09:51:40.000Z,
  mtime: 2016-11-22T09:51:40.000Z,
  ctime: 2016-11-22T09:51:40.000Z,
  birthtime: 2016-11-22T09:51:40.000Z }
  SOLINK_MODULE(target) Release/fse.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9

> websocket@1.0.23 install /Users/kamsties/Code/mdwiki/node_modules/websocket
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
mdwiki@0.1.0 /Users/kamsties/Code/mdwiki
├── bower@1.8.0 
├─┬ grunt@1.0.1 
│ ├── coffee-script@1.10.0 
│ ├─┬ dateformat@1.0.12 
│ │ ├── get-stdin@4.0.1 
│ │ └─┬ meow@3.7.0 
│ │   ├── camelcase-keys@2.1.0 
│ │   ├── decamelize@1.2.0 
│ │   ├─┬ loud-rejection@1.6.0 
│ │   │ ├─┬ currently-unhandled@0.4.1 
│ │   │ │ └── array-find-index@1.0.2 
│ │   │ └── signal-exit@3.0.1 
│ │   ├── map-obj@1.0.1 
│ │   ├─┬ normalize-package-data@2.3.5 
│ │   │ ├── hosted-git-info@2.1.5 
│ │   │ ├─┬ is-builtin-module@1.0.0 
│ │   │ │ └── builtin-modules@1.1.1 
│ │   │ └─┬ validate-npm-package-license@3.0.1 
│ │   │   ├─┬ spdx-correct@1.0.2 
│ │   │   │ └── spdx-license-ids@1.2.2 
│ │   │   └── spdx-expression-parse@1.0.4 
│ │   ├─┬ read-pkg-up@1.0.1 
│ │   │ └─┬ read-pkg@1.1.0 
│ │   │   ├─┬ load-json-file@1.1.0 
│ │   │   │ └── pify@2.3.0 
│ │   │   └── path-type@1.1.0 
│ │   ├─┬ redent@1.0.0 
│ │   │ ├── indent-string@2.1.0 
│ │   │ └── strip-indent@1.0.1 
│ │   └── trim-newlines@1.0.0 
│ ├── eventemitter2@0.4.14 
│ ├── exit@0.1.2 
│ ├─┬ findup-sync@0.3.0 
│ │ └── glob@5.0.15 
│ ├─┬ glob@7.0.6 
│ │ ├── fs.realpath@1.0.0 
│ │ ├─┬ inflight@1.0.6 
│ │ │ └── wrappy@1.0.2 
│ │ ├── inherits@2.0.3 
│ │ └── once@1.4.0 
│ ├── grunt-known-options@1.1.0 
│ ├─┬ grunt-legacy-log@1.0.0 
│ │ ├── colors@1.1.2 
│ │ ├─┬ grunt-legacy-log-utils@1.0.0 
│ │ │ └── lodash@4.3.0 
│ │ ├── hooker@0.2.3 
│ │ └── underscore.string@3.2.3 
│ ├─┬ grunt-legacy-util@1.0.0 
│ │ ├── getobject@0.1.0 
│ │ ├── lodash@4.3.0 
│ │ └─┬ which@1.2.12 
│ │   └── isexe@1.1.2 
│ ├── iconv-lite@0.4.15 
│ ├─┬ js-yaml@3.5.5 
│ │ ├─┬ argparse@1.0.9 
│ │ │ └── sprintf-js@1.0.3 
│ │ └── esprima@2.7.3 
│ ├─┬ minimatch@3.0.3 
│ │ └─┬ brace-expansion@1.1.6 
│ │   ├── balanced-match@0.4.2 
│ │   └── concat-map@0.0.1 
│ ├─┬ nopt@3.0.6 
│ │ └── abbrev@1.0.9 
│ ├── path-is-absolute@1.0.1 
│ └── rimraf@2.2.8 
├─┬ grunt-cli@1.2.0 
│ └── resolve@1.1.7 
├─┬ grunt-contrib-concat@1.0.1 
│ ├─┬ chalk@1.1.3 
│ │ ├── ansi-styles@2.2.1 
│ │ ├── escape-string-regexp@1.0.5 
│ │ ├─┬ has-ansi@2.0.0 
│ │ │ └── ansi-regex@2.0.0 
│ │ ├── strip-ansi@3.0.1 
│ │ └── supports-color@2.0.0 
│ └── source-map@0.5.6 
├─┬ grunt-contrib-copy@1.0.0 
│ └── file-sync-cmp@0.1.1 
├─┬ grunt-contrib-less@1.4.0 
│ ├── async@2.1.2 
│ ├─┬ less@2.7.1 
│ │ ├─┬ errno@0.1.4 
│ │ │ └── prr@0.0.0 
│ │ ├── graceful-fs@4.1.10 
│ │ ├── image-size@0.5.0 
│ │ ├── mime@1.3.4 
│ │ ├─┬ mkdirp@0.5.1 
│ │ │ └── minimist@0.0.8 
│ │ └─┬ promise@7.1.1 
│ │   └── asap@2.0.5 
│ └── lodash@4.17.2 
├─┬ grunt-contrib-uglify@1.0.2 
│ ├── lodash@4.17.2 
│ ├─┬ maxmin@1.1.0 
│ │ ├── figures@1.7.0 
│ │ ├─┬ gzip-size@1.0.0 
│ │ │ ├─┬ browserify-zlib@0.1.4 
│ │ │ │ └── pako@0.2.9 
│ │ │ └─┬ concat-stream@1.5.2 
│ │ │   ├─┬ readable-stream@2.0.6 
│ │ │   │ ├── process-nextick-args@1.0.7 
│ │ │   │ └── util-deprecate@1.0.2 
│ │ │   └── typedarray@0.0.6 
│ │ └── pretty-bytes@1.0.4 
│ ├─┬ uglify-js@2.6.4 
│ │ ├── async@0.2.10 
│ │ ├── uglify-to-browserify@1.0.2 
│ │ └─┬ yargs@3.10.0 
│ │   ├── camelcase@1.2.1 
│ │   ├─┬ cliui@2.1.0 
│ │   │ ├─┬ center-align@0.1.3 
│ │   │ │ ├─┬ align-text@0.1.4 
│ │   │ │ │ ├── longest@1.0.1 
│ │   │ │ │ └── repeat-string@1.6.1 
│ │   │ │ └── lazy-cache@1.0.4 
│ │   │ └── right-align@0.1.3 
│ │   └── window-size@0.1.0 
│ └── uri-path@1.0.0 
├─┬ grunt-contrib-watch@1.0.0 
│ ├── async@1.5.2 
│ ├─┬ gaze@1.1.2 
│ │ └─┬ globule@1.1.0 
│ │   ├── glob@7.1.1 
│ │   └── lodash@4.16.6 
│ ├── lodash@3.10.1 
│ └─┬ tiny-lr@0.2.1 
│   ├─┬ body-parser@1.14.2 
│   │ ├── bytes@2.2.0 
│   │ ├── content-type@1.0.2 
│   │ ├── depd@1.1.0 
│   │ ├─┬ http-errors@1.3.1 
│   │ │ └── statuses@1.3.1 
│   │ ├── iconv-lite@0.4.13 
│   │ ├─┬ on-finished@2.3.0 
│   │ │ └── ee-first@1.1.1 
│   │ ├── qs@5.2.0 
│   │ ├─┬ raw-body@2.1.7 
│   │ │ ├── bytes@2.4.0 
│   │ │ ├── iconv-lite@0.4.13 
│   │ │ └── unpipe@1.0.0 
│   │ └─┬ type-is@1.6.14 
│   │   ├── media-typer@0.3.0 
│   │   └─┬ mime-types@2.1.13 
│   │     └── mime-db@1.25.0 
│   ├─┬ debug@2.2.0 
│   │ └── ms@0.7.1 
│   ├─┬ faye-websocket@0.10.0 
│   │ └─┬ websocket-driver@0.6.5 
│   │   └── websocket-extensions@0.1.1 
│   ├── livereload-js@2.2.2 
│   ├── parseurl@1.3.1 
│   └── qs@5.1.0 
├─┬ grunt-http-server@2.0.0 
│ ├─┬ divhide@2.0.1 
│ │ └── lodash@3.9.3 
│ ├─┬ http-server@0.9.0 
│ │ ├── colors@1.0.3 
│ │ ├── corser@2.0.1 
│ │ ├─┬ ecstatic@1.4.1 
│ │ │ ├── he@0.5.0 
│ │ │ └── url-join@1.1.0 
│ │ ├─┬ http-proxy@1.15.2 
│ │ │ ├── eventemitter3@1.2.0 
│ │ │ └── requires-port@1.0.0 
│ │ ├─┬ portfinder@0.4.0 
│ │ │ └── async@0.9.0 
│ │ └─┬ union@0.4.6 
│ │   └── qs@2.3.3 
│ ├── lodash@4.2.1 
│ ├── opener@1.4.1 
│ └─┬ showdown@1.5.0 
│   └─┬ yargs@3.32.0 
│     ├── camelcase@2.1.1 
│     ├─┬ cliui@3.2.0 
│     │ └── wrap-ansi@2.0.0 
│     ├─┬ os-locale@1.4.0 
│     │ └─┬ lcid@1.0.0 
│     │   └── invert-kv@1.0.0 
│     ├── window-size@0.1.4 
│     └── y18n@3.2.1 
├─┬ grunt-reload@0.4.0  (git://github.com/webxl/grunt-reload.git#15d96e8c46d15a1c89060527434d708a1d80c359)
│ ├── buffers@0.1.1 
│ ├─┬ connect@2.3.9 
│ │ ├── bytes@0.1.0 
│ │ ├── cookie@0.0.4 
│ │ ├── crc@0.2.0 
│ │ ├── formidable@1.0.11 
│ │ ├── fresh@0.1.0 
│ │ ├── qs@0.4.2 
│ │ └─┬ send@0.0.3 
│ │   ├── mime@1.2.6 
│ │   └── range-parser@0.0.4 
│ ├─┬ grunt@0.4.0-rc4 
│ │ ├── async@0.1.22 
│ │ ├── coffee-script@1.3.3 
│ │ ├── colors@0.6.2 
│ │ ├── dateformat@1.0.2-1.2.3 
│ │ ├─┬ findup-sync@0.1.3 
│ │ │ ├─┬ glob@3.2.11 
│ │ │ │ └── minimatch@0.3.0 
│ │ │ └── lodash@2.4.2 
│ │ ├─┬ glob@3.1.21 
│ │ │ ├── graceful-fs@1.2.3 
│ │ │ └── inherits@1.0.2 
│ │ ├── iconv-lite@0.2.11 
│ │ ├─┬ js-yaml@1.0.3 
│ │ │ └─┬ argparse@0.1.16 
│ │ │   ├── underscore@1.7.0 
│ │ │   └── underscore.string@2.4.0 
│ │ ├── lodash@0.9.2 
│ │ ├─┬ minimatch@0.2.14 
│ │ │ ├── lru-cache@2.7.3 
│ │ │ └── sigmund@1.0.1 
│ │ ├── nopt@1.0.10 
│ │ ├─┬ rimraf@2.0.3 
│ │ │ └── graceful-fs@1.1.14 
│ │ ├── underscore.string@2.2.1 
│ │ └── which@1.0.9 
│ ├─┬ http-proxy@0.8.0 
│ │ ├── optimist@0.2.8 
│ │ └── pkginfo@0.2.3 
│ ├─┬ request@2.12.0 
│ │ ├─┬ form-data@0.0.3 
│ │ │ ├── async@0.1.9 
│ │ │ └─┬ combined-stream@0.0.3 
│ │ │   └── delayed-stream@0.0.5 
│ │ └── mime@1.2.7 
│ └─┬ websocket@1.0.23 
│   ├── nan@2.4.0 
│   ├─┬ typedarray-to-buffer@3.1.2 
│   │ └── is-typedarray@1.0.0 
│   └── yaeti@0.0.4 
├─┬ grunt-shell@1.3.1 
│ ├─┬ npm-run-path@1.0.0 
│ │ └── path-key@1.0.0 
│ └── object-assign@4.1.0 
├─┬ grunt-ts@5.5.1 
│ ├─┬ chokidar@1.0.6 
│ │ ├─┬ anymatch@1.3.0 
│ │ │ └─┬ micromatch@2.3.11 
│ │ │   ├─┬ arr-diff@2.0.0 
│ │ │   │ └── arr-flatten@1.0.1 
│ │ │   ├── array-unique@0.2.1 
│ │ │   ├─┬ braces@1.8.5 
│ │ │   │ ├─┬ expand-range@1.8.2 
│ │ │   │ │ └─┬ fill-range@2.2.3 
│ │ │   │ │   ├── is-number@2.1.0 
│ │ │   │ │   └── randomatic@1.1.5 
│ │ │   │ ├── preserve@0.2.0 
│ │ │   │ └── repeat-element@1.1.2 
│ │ │   ├─┬ expand-brackets@0.1.5 
│ │ │   │ └── is-posix-bracket@0.1.1 
│ │ │   ├── extglob@0.3.2 
│ │ │   ├── filename-regex@2.0.0 
│ │ │   ├── is-extglob@1.0.0 
│ │ │   ├── is-glob@2.0.1 
│ │ │   ├─┬ kind-of@3.0.4 
│ │ │   │ └── is-buffer@1.1.4 
│ │ │   ├── normalize-path@2.0.1 
│ │ │   ├─┬ object.omit@2.0.1 
│ │ │   │ ├─┬ for-own@0.1.4 
│ │ │   │ │ └── for-in@0.1.6 
│ │ │   │ └── is-extendable@0.1.1 
│ │ │   ├─┬ parse-glob@3.0.4 
│ │ │   │ ├─┬ glob-base@0.3.0 
│ │ │   │ │ ├── glob-parent@2.0.0 
│ │ │   │ │ └── is-glob@2.0.1 
│ │ │   │ ├── is-dotfile@1.0.2 
│ │ │   │ └── is-glob@2.0.1 
│ │ │   └─┬ regex-cache@0.4.3 
│ │ │     ├── is-equal-shallow@0.1.3 
│ │ │     └── is-primitive@2.0.0 
│ │ ├── async-each@0.1.6 
│ │ ├── fsevents@0.3.8 
│ │ ├─┬ glob-parent@1.3.0 
│ │ │ └── is-glob@2.0.1 
│ │ ├─┬ is-binary-path@1.0.1 
│ │ │ └── binary-extensions@1.7.0 
│ │ ├── is-glob@1.1.3 
│ │ └─┬ readdirp@1.4.0 
│ │   ├── minimatch@0.2.14 
│ │   └─┬ readable-stream@1.0.34 
│ │     ├── core-util-is@1.0.2 
│ │     ├── isarray@0.0.1 
│ │     └── string_decoder@0.10.31 
│ ├─┬ csproj2ts@0.0.7 
│ │ ├── es6-promise@2.3.0 
│ │ └─┬ xml2js@0.4.17 
│ │   ├── sax@1.2.1 
│ │   └─┬ xmlbuilder@4.2.1 
│ │     └── lodash@4.17.2 
│ ├── es6-promise@0.1.2 
│ ├── lodash@2.4.1 
│ ├── ncp@0.5.1 
│ ├── rimraf@2.2.6 
│ ├── semver@5.3.0 
│ ├─┬ strip-bom@2.0.0 
│ │ └── is-utf8@0.2.1 
│ ├── typescript@1.8.9 
│ ├── underscore@1.5.1 
│ └── underscore.string@2.3.3 
├─┬ handlebars@4.0.6 
│ ├─┬ optimist@0.6.1 
│ │ ├── minimist@0.0.10 
│ │ └── wordwrap@0.0.2 
│ └─┬ source-map@0.4.4 
│   └── amdefine@1.0.1 
├─┬ load-grunt-tasks@3.5.2 
│ ├── arrify@1.0.1 
│ ├─┬ multimatch@2.1.0 
│ │ ├── array-differ@1.0.0 
│ │ └── array-union@1.0.2 
│ ├─┬ pkg-up@1.0.0 
│ │ └─┬ find-up@1.1.2 
│ │   ├── path-exists@2.1.0 
│ │   └─┬ pinkie-promise@2.0.1 
│ │     └── pinkie@2.0.4 
│ └─┬ resolve-pkg@0.1.0 
│   └── resolve-from@2.0.0 
├── typescript@1.8.10 
└─┬ typings@1.5.0 
  ├── any-promise@1.3.0 
  ├── archy@1.0.0 
  ├── bluebird@3.4.6 
  ├─┬ cli-truncate@0.2.1 
  │ ├── slice-ansi@0.0.4 
  │ └─┬ string-width@1.0.2 
  │   ├── code-point-at@1.1.0 
  │   └─┬ is-fullwidth-code-point@1.0.0 
  │     └── number-is-nan@1.0.1 
  ├─┬ columnify@1.5.4 
  │ └─┬ wcwidth@1.0.1 
  │   └─┬ defaults@1.0.3 
  │     └── clone@1.0.2 
  ├── elegant-spinner@1.0.1 
  ├── has-unicode@2.0.1 
  ├── listify@1.0.0 
  ├─┬ log-update@1.0.2 
  │ ├── ansi-escapes@1.4.0 
  │ └─┬ cli-cursor@1.0.2 
  │   └─┬ restore-cursor@1.0.1 
  │     ├── exit-hook@1.1.1 
  │     └── onetime@1.1.0 
  ├── minimist@1.2.0 
  ├── promise-finally@2.2.1 
  ├─┬ typings-core@1.6.1 
  │ ├── array-uniq@1.0.3 
  │ ├─┬ configstore@2.1.0 
  │ │ ├─┬ dot-prop@3.0.0 
  │ │ │ └── is-obj@1.0.1 
  │ │ ├── os-tmpdir@1.0.2 
  │ │ ├── osenv@0.1.3 
  │ │ ├── uuid@2.0.3 
  │ │ └─┬ write-file-atomic@1.2.0 
  │ │   ├── imurmurhash@0.1.4 
  │ │   └── slide@1.1.6 
  │ ├─┬ detect-indent@4.0.0 
  │ │ └─┬ repeating@2.0.1 
  │ │   └── is-finite@1.0.2 
  │ ├─┬ has@1.0.1 
  │ │ └── function-bind@1.1.0 
  │ ├─┬ invariant@2.2.2 
  │ │ └─┬ loose-envify@1.3.0 
  │ │   └── js-tokens@2.0.0 
  │ ├─┬ is-absolute@0.2.6 
  │ │ ├─┬ is-relative@0.2.1 
  │ │ │ └─┬ is-unc-path@0.1.1 
  │ │ │   └── unc-path-regex@0.1.2 
  │ │ └── is-windows@0.2.0 
  │ ├── lockfile@1.0.2 
  │ ├─┬ make-error-cause@1.2.2 
  │ │ └── make-error@1.2.1 
  │ ├─┬ object.pick@1.2.0 
  │ │ └─┬ isobject@2.1.0 
  │ │   └── isarray@1.0.0 
  │ ├─┬ parse-json@2.2.0 
  │ │ └─┬ error-ex@1.3.0 
  │ │   └── is-arrayish@0.2.1 
  │ ├─┬ popsicle@8.2.0 
  │ │ ├─┬ form-data@2.1.2 
  │ │ │ ├── asynckit@0.4.0 
  │ │ │ └─┬ combined-stream@1.0.5 
  │ │ │   └── delayed-stream@1.0.0 
  │ │ ├── throwback@1.1.1 
  │ │ └─┬ tough-cookie@2.3.2 
  │ │   └── punycode@1.4.1 
  │ ├─┬ popsicle-proxy-agent@3.0.0 
  │ │ ├─┬ http-proxy-agent@1.0.0 
  │ │ │ ├─┬ agent-base@2.0.1 
  │ │ │ │ └── semver@5.0.3 
  │ │ │ └── extend@3.0.0 
  │ │ └── https-proxy-agent@1.0.0 
  │ ├── popsicle-retry@3.2.1 
  │ ├── popsicle-rewrite@1.0.0 
  │ ├── popsicle-status@2.0.0 
  │ ├─┬ rc@1.1.6 
  │ │ ├── deep-extend@0.4.1 
  │ │ ├── ini@1.3.4 
  │ │ └── strip-json-comments@1.0.4 
  │ ├── rimraf@2.5.4 
  │ ├─┬ sort-keys@1.1.2 
  │ │ └── is-plain-obj@1.1.0 
  │ ├── string-template@1.0.0 
  │ ├── thenify@3.2.1 
  │ ├── throat@3.0.0 
  │ ├─┬ touch@1.0.0 
  │ │ └── nopt@1.0.10 
  │ ├── typescript@2.0.10 
  │ └── zip-object@0.1.0 
  ├─┬ update-notifier@1.0.2 
  │ ├─┬ boxen@0.6.0 
  │ │ ├── ansi-align@1.1.0 
  │ │ ├── cli-boxes@1.0.0 
  │ │ ├── filled-array@1.1.0 
  │ │ └── widest-line@1.0.0 
  │ ├── is-npm@1.0.0 
  │ ├─┬ latest-version@2.0.0 
  │ │ └─┬ package-json@2.4.0 
  │ │   ├─┬ got@5.7.1 
  │ │   │ ├─┬ create-error-class@3.0.2 
  │ │   │ │ └── capture-stack-trace@1.0.0 
  │ │   │ ├── duplexer2@0.1.4 
  │ │   │ ├── is-redirect@1.0.0 
  │ │   │ ├── is-retry-allowed@1.1.0 
  │ │   │ ├── is-stream@1.1.0 
  │ │   │ ├── lowercase-keys@1.0.0 
  │ │   │ ├── node-status-codes@1.0.0 
  │ │   │ ├── read-all-stream@3.1.0 
  │ │   │ ├── timed-out@3.0.0 
  │ │   │ ├── unzip-response@1.0.2 
  │ │   │ └─┬ url-parse-lax@1.0.0 
  │ │   │   └── prepend-http@1.0.4 
  │ │   ├── registry-auth-token@3.1.0 
  │ │   └── registry-url@3.1.0 
  │ ├── lazy-req@1.1.0 
  │ ├── semver-diff@2.1.0 
  │ └─┬ xdg-basedir@2.0.0 
  │   └── os-homedir@1.0.2 
  ├── wordwrap@1.0.0 
  └── xtend@4.0.1 

npm WARN mdwiki@0.1.0 No repository field.
npm WARN mdwiki@0.1.0 No license field.
./node_modules/.bin/bower install
bower cached        https://github.com/twbs/bootstrap.git#3.0.0
bower validate      3.0.0 against https://github.com/twbs/bootstrap.git#3.0.0
bower cached        https://github.com/jquery/jquery-dist.git#1.8.3
bower validate      1.8.3 against https://github.com/jquery/jquery-dist.git#1.8.3
bower cached        https://github.com/jquery/jquery-dist.git#3.1.1
bower validate      3.1.1 against https://github.com/jquery/jquery-dist.git#>= 1.9.0

Please note that,
    mdwiki depends on jquery#1.8.3 which resolved to jquery#1.8.3
    bootstrap#3.0.0 depends on jquery#>= 1.9.0 which resolved to jquery#3.1.1
Resort to using jquery#1.8.3 which resolved to jquery#1.8.3
Code incompatibilities may occur.

bower install       jquery#1.8.3
bower install       bootstrap#3.0.0

jquery#1.8.3 bower_components/jquery

bootstrap#3.0.0 bower_components/bootstrap
└── jquery#1.8.3
./node_modules/.bin/typings install

├── handlebars (global)
└── jquery (global)

./node_modules/.bin/grunt debug release
Running "ts:base" (ts) task
Compiling...
Fast compile will not work when --out is specified. Ignoring fast compilation
Using tsc v1.8.10



TypeScript compilation complete: 1.22s for 16 TypeScript files.

Running "less:dev" (less) task
>> 1 stylesheet created.

Running "shell:compile_templates" (shell) task

Running "concat:dev" (concat) task

Running "index_debug" task
Building single index.html of type: debug
Generated 'dist/mdwiki-debug.html' from 'index.tmpl'

Running "ts:base" (ts) task
Compiling...
Fast compile will not work when --out is specified. Ignoring fast compilation
Using tsc v1.8.10



TypeScript compilation complete: 1.19s for 16 TypeScript files.

Running "less:min" (less) task
>> 1 stylesheet created.

Running "shell:compile_templates" (shell) task

Running "concat:dev" (concat) task

Running "uglify:dist" (uglify) task
File tmp/MDwiki.min.js created: 109.7 kB → 49.91 kB
>> 1 file created.

Running "index" task
Building single index.html of type: release
Generated 'dist/mdwiki.html' from 'index.tmpl'

Done.
```

**Comment by Dynalon** [comment]
Duplicate of #259 and #194 -> basically MDwiki master branch is broken for quite a while and needs fixing first.

**Comment by bidspells** [comment]
I see... I cloned the the 0.6.x branch and successfully complied mdwiki using the 6.9.1 LTS version of node.js (following the procedure described on the page of this branch).

Interestingly, this solved my second issue. Now, Safari is able to instantly load mdwiki without waiting 20-30 seconds at the first load of mdwiki.html itself (this behavior was reported also in issue #198).  

Really cool!!
